tests/i18n/tests.py

```diff
@@ -1424,8 +1424,17 @@ def test_check_for_language(self):
         self.assertTrue(check_for_language('en'))
         self.assertTrue(check_for_language('en-us'))
         self.assertTrue(check_for_language('en-US'))
+        self.assertTrue(check_for_language('be'))
+        self.assertTrue(check_for_language('be@latin'))
+        self.assertTrue(check_for_language('sr-RS@latin'))
+        self.assertTrue(check_for_language('sr-RS@12345'))
```

Comment:
Anyway I don't think that hardcoding the list is good idea, here is list of variants available on my system:

```
sed -n '/@/ s/.*@\(.*\) .*/\1/p' /usr/share/i18n/SUPPORTED | sort -u
abegede
cyrillic
devanagari
euro
iqtelif
latin
saaho
valencia
```